Frontend/web/src/styles/globals.css

```
@@ -39,10 +46,12 @@ body {
 
 @layer base {
   body {
-    @apply bg-slate-50 text-slate-900 antialiased;
+    @apply text-slate-900 antialiased;
     background:
-      radial-gradient(circle at top right, rgba(37, 99, 235, 0.08), transparent 28rem),
-      linear-gradient(180deg, #f8fafc 0%, #eef2f7 100%);
+      radial-gradient(900px circle at 80% -10%, rgba(228, 168, 83, 0.06) 0%, transparent 55%),
+      radial-gradient(900px circle at 20% -10%, rgba(37, 99, 235, 0.05) 0%, transparent 55%),
+      var(--page-bg);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)